src/concrete/ml/pytest/torch_models.py

```diff
@@ -781,47 +823,6 @@ def forward(self, x):
         x = self.fc1(x)
         return x
 
-    def test_torch(self, test_loader):
```

Comment:
why was this removed ?

Reply:
it was converted to a simple function instead of a class method. Thus it can be used on other model instances